Assets/Scripts/PickupHead.cs

```diff
@@ -7,15 +7,18 @@ public class PickupHead : MonoBehaviour {
 	private GameObject body;
 	private GameObject head;
 	private HeadController headController;
+	private Jump jumpScript;
 	// Use this for initialization
 	void Start () {
 		body = this.gameObject;
 		head = GameObject.Find ("Head");
 		headController = head.GetComponent<HeadController> ();
+		jumpScript = this.GetComponent<Jump> ();
 	}
 
 	void OnCollisionEnter(Collision collision) {
-		if (collision.gameObject.tag == "HeadIsDetached") {
+		bool isGrounded = jumpScript.isGrounded();
+		if (collision.gameObject.tag == "HeadIsDetached" && isGrounded) {
```

Comment:
Did you test out flinging while jumping? It sounds like we would want to include that, instead of requiring that the player is on the ground before being able to fling. What do you think?

Reply:
Yes, that is the reason I had to check for grounded in PickupHead. The player can jump and throw at the moment.

Reply:
Sorry. I didn't look at the context. I thought this was about flinging, not picking up.